### DISCUSSION TITLE
Help in getting my student benefit

### DISCUSSION THREAD

**Discussion by ghost** [community/community#55954] [GitHub Education]
### Hi everyone,

Hello guys, I am an online student at IU Internationale Hochschule.
I was trying to apply for student benefits but got rejected because I am not near my college.

I have a contract from IU, I also have my student ID, but each time I try, I get the same message, **_your location and college location is mismatched_** and if you are an online student use **academic affiliation documents**.

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
